Riot/Modules/SplitView/SplitViewCoordinator.swift

```diff
@@ -93,8 +93,7 @@ final class SplitViewCoordinator: NSObject, SplitViewCoordinatorType {
             self.splitViewController.delegate = self
             
             // Create primary controller
-            let tabBarCoordinator = self.createTabBarCoordinator()
-            tabBarCoordinator.delegate = self
+            let tabBarCoordinator: TabBarCoordinatorType = BuildSettings.isNewAppLayoutActivated ? self.createAllChatsCoordinator() : self.createTabBarCoordinator()
```

Comment:
We should rename this type, doesn't really make sense anymore, does it?

Reply:
I made the change and also created a protocol for the `MasterTabBarViewController` and the `AllChatsViewController` to make the change safer.

Reply:
Nice, looking good! 👍